src/main/java/org/jabref/logic/importer/fileformat/BibtexParser.java

```diff
@@ -386,7 +390,23 @@ private void parseJabRefComment(Map<String, String> meta) {
             } catch (ParseException ex) {
                 parserResult.addException(ex);
             }
+        } else if (comment.substring(0, Math.min(comment.length(), MetaData.META_FLAG_VERSION_010.length())).equals(MetaData.META_FLAG_VERSION_010)) {
```

Comment:
Java strings support `startsWith`. I think, this should be used here instead of this complicated thign.